staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -81,8 +81,32 @@ type HorizontalPodAutoscalerSpec struct {
 	// If not set, the default HPAScalingRules for scale up and scale down are used.
 	// +optional
 	Behavior *HorizontalPodAutoscalerBehavior `json:"behavior,omitempty" protobuf:"bytes,5,opt,name=behavior"`
+
+	// UpdatePolicy describes how changes are applied to the cluster.
+	// If not specified, all values are set to default
+	UpdatePolicy *HorizontalPodAutoscalerUpdatePolicy `json:"updatePolicy,omitempty" protobuf:"bytes,6,opt,name=updatePolicy"`
```

Comment:
Please do not add new fields to the v2beta2 API as it is [deprecated](https://github.com/kubernetes/kubernetes/blob/393337dc84a90979f23f0cf11520bf1adb995a03/staging/src/k8s.io/api/autoscaling/v2beta2/types.go#L30).  You can just add this new field directly to the v2 (stable) API alone.